test_list_files_multi fails with python-3.3.2

```
======================================================================
FAIL: test_list_files_multi (tests.CommandLineOutputTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/var/package-manager/tmp/portage/net-p2p/torrentinfo-1.8.6/work/torrentinfo-1.8.6/test/tests.py", line 609, in test_list_files_multi
    self.assertEqual(self.out.getvalue(), return_string)
AssertionError: 'multi_bytes.torrent\n    files    \n        0\n            length    \n         [truncated]... != 'multi_bytes.torrent\n    files    \n        0\n            path    \n           [truncated]...
  multi_bytes.torrent
      files    
          0
+             path    
+                 megabyte
              length    
                  1048576
+         1
              path    
-                 megabyte
+                 two_megabytes
?                 ++++        +
-         1
              length    
                  2097152
-             path    
-                 two_megabytes
      piece length    
              262144
      pieces    
              [240 UTF-8 Bytes]



----------------------------------------------------------------------
Ran 87 tests in 0.549s

FAILED (failures=1)

```

--- a/src/torrentinfo.py
+++ b/src/torrentinfo.py
@@ -569,7 +569,7 @@ def list_files(config, torrent, detailed=False):
 
             config.formatter.string_format(TextFormatter.NORMAL, config, '\n')
             if detailed:
-                for kwrd in filestorrent[index]:
+                for kwrd in sorted(filestorrent[index], reverse=True):
                     start_line(config, kwrd, 3, postfix='\n')
                     dump(filestorrent[index][kwrd], config, 4)
             else:

